### PR TITLE
Add Cura Lulzbot Edition v21.00

### PR DESCRIPTION
**Pull request by wastrachan**
_If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so._

After making all changes to the cask:
- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:
- [X] Named the cask according to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [X] Checked that the cask was not already refused in [closed issues](https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed).

### CODE REVIEW

**Comment by miccal** [comment]
You can use the `version` in the `url`, like so:
```
url 'http://download.alephobjects.com/lulzbot/Software/Cura/Packages/Cura_#{version}/cura_#{version}-b796.dmg'
```

**Comment by wastrachan** [comment]
I noticed that in the contribution guide, but I fear that the `-b796` in `cura_#{version}-b796` is version specific, and will need to be updated on new releases.

**Comment by miccal** [comment]
Maybe change `version` to `21.00-b796`, then the url can be written
```
url "http://download.alephobjects.com/lulzbot/Software/Cura/Packages/Cura_#{version.major_minor}/cura_#{version}-b796.dmg"
```

**Comment by wastrachan** [comment]
Great suggestion, wasn't aware about `version.major_minor`. Updated.

File: Casks/cura-lulzbot.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask 'cura-lulzbot' do
```
Comment (vitorgalvao) [inline Casks/cura-lulzbot.rb]: This must be `lulzbot-cura`, instead. See “[Forks and Apps with Conflicting Names](https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#forks-and-apps-with-conflicting-names)”.

I’ll make the change.

**Pull request merged**